index.js

```diff
@@ -68,20 +90,81 @@ function Chapter (index, chapterName, fileName, content) {
   return {index: index, chapterName: chapterName, fileName: fileName, content: content}
 }
 
+Streampub.prototype._generateCover = function(async) {
```

Comment:
Sooo, two things…

`async` is likely to be a keyword [Real Soon Now](https://github.com/tc39/ecmascript-asyncawait), so it shouldn't be a variable name.

And I'd would prefer this not be functionality that's controlled by a flag. I'd prefer two functions, perhaps with one calling the other.